app/src/main/java/org/thoughtcrime/securesms/database/SmsDatabase.java

```diff
@@ -633,6 +633,7 @@ public boolean deleteMessage(long messageId) {
     SQLiteDatabase db = databaseHelper.getWritableDatabase();
     long threadId = getThreadIdForMessage(messageId);
     db.delete(TABLE_NAME, ID_WHERE, new String[] {messageId+""});
+    notifyConversationListeners(threadId);
```

Comment:
so I guess this is the fix essentially.

It looks more in line with `MmsDatabase#deleteMessage` now. And I guess those stickerListeners in that one are mms only. so LGTM.